kratos/python/add_properties_to_python.cpp

```diff
@@ -177,6 +227,16 @@ void  AddPropertiesToPython(pybind11::module& m)
     .def("HasVariables", &Properties::HasVariables)
     .def("HasTables", &Properties::HasTables)
     .def("IsEmpty", &Properties::IsEmpty)
+    .def("NumberOfSubproperties", &Properties::NumberOfSubproperties)
+    .def("AddSubProperty", &Properties::AddSubProperty)
+    .def("GetSubProperty", GetSubProperty1)
+    .def("HasSubPropertiesById", HasSubPropertiesById1)
+    .def("GetSubPropertiesById", GetSubPropertiesById1)
+    .def("GetSubPropertiesByAddress", GetSubPropertiesByAddress1)
+    .def("HasSubPropertiesByAddress", HasSubPropertiesByAddress1)
+    .def("GetSubProperties", GetSubProperties1)
+    .def("GetSubProperties", GetSubProperties2)
+    .def("SetSubProperties", &Properties::SetSubProperties)
```

Comment:
we should be careful what we want to expose to python, there is currently the same discussion for nodes/elems/conds => #2181 & #5001 
(I am not sure it is the same problems, just disregard this comment if not applicable)

Reply:
@philbucher i think this is ok now. Please give your green light if you agree